### DISCUSSION TITLE
slight problem with custom colours

### DISCUSSION THREAD

**Discussion by spaciousmind** [ScratchAddons/ScratchAddons#2893] [Help]
Hi there, I think this is to do with custom colours --- i have a problem with the colours only when I click on a block in the editor to show me the result (such as an "item (whatever) of list" or an operator block like "2+2") it shows the result in black  (or very dark text)  on a coloured background - almost unreadable. All my other text is set to white so I can read it on the colour blocks. 

I'd be happy if it just showed the results in white text or either black text on a white background.

any solution? is this a bug?

**Comment by WorldLanguages** [comment]
A screenshot of the bug and your custom colors addon settings would be useful

**Reply by spaciousmind** [reply]
![screenshot scratch](https://user-images.githubusercontent.com/60484647/124243115-91de8980-db71-11eb-831e-822317157508.jpg)

Oh yeah. here it is.... check the three examples at the top. 
the colour thing that displays the answer seems to change at random. at first it was green, then i clicked on the list block and it changed them to red (you can see in the screenshots) then i refreshed the page and it displayed it as black. All of which are hard to read, although it's worse when it's on green. 
The colours wouldn't be a problem if that text was white.

There's my custom colour settings and also dark mode settings in case it's a conflict between the two

**Reply by CST1229** [reply]
I already reported this in #2553. It's based on the last dropdown you clicked.

**Reply by spaciousmind** [reply]
ah yes that's the one. thanks CST1229 I'll subscribe to your thread
